runtime/compiler/control/JITaaSCompilationThread.hpp

```diff
@@ -57,6 +57,7 @@ class ClientSessionData
       TR_OpaqueClassBlock * leafComponentClass;
       void *classLoader;
       TR_OpaqueClassBlock * hostClass;
+      TR_OpaqueClassBlock * componentClass;
```

Comment:
Because the name of this field is not the same as the name of the field in the j9class I would like a comment showing what exactly it is caching (J9ArrayClass::componentType) and the fact that is applicable only for array classes